src/app/edit-absences/components/edit-absences-edit/edit-absences-edit.component.ts

```diff
@@ -118,6 +119,21 @@ export class EditAbsencesEditComponent implements OnInit, OnDestroy {
         .pipe(takeUntil(this.destroy$))
         .subscribe(this.updateAbsenceTypeIdDisabled.bind(this));
     }
+
+    // Remove Category HalfDay if the corresponding PresenceType is inactive
+    this.absenceTypes$
+      .pipe(
+        map((types) =>
+          types.filter((t) => isHalfDay(t, this.settings) && t.Active)
```

Comment:
Micro-Improvement: hier könntest du ein `Boolean(types.find((t) => isHalfDay(t, this.settings))?.Active)` machen und gleich auf einen `boolean` mappen. Das hat auch zum Vorteil, dass das Iterieren abgebrochen wir, sobald der Halfday gefunden wird.